src/Seq.App.Slack/SlackReactor.cs

```diff
@@ -67,14 +67,29 @@ public class SlackReactor : Reactor, ISubscribeTo<LogEventData>
             HelpText = "If a property when converted to a string is longer than this number it will be truncated")]
         public int? MaxPropertyLength { get; set; } = null;
 
+        [SeqAppSetting(
+            DisplayName = "Included properties",
+            IsOptional = true,
+            HelpText = "Comma separated list (no spaces) of properties to include as attachments.  The default is to include all properties.")]
```

Comment:
Could we `Trim()` the property values just to make this a little more user-friendly?

Reply:
Good idea.  Certainly more user friendly and easy to achieve.

Reply:
This is my first contribution to an open source project so not sure of etiquette.  Do I resolve these conversations or would you do it @nblumhardt, as the person who raised the original comment?
Thanks